client.go

```diff
@@ -173,7 +182,7 @@ type ovndb struct {
 	disconnectCB OVNDisconnectedCallback
 }
 
-func NewClient(cfg *Config) (Client, error) {
+func NewClient(cfg *Config, db string) (Client, error) {
```

Comment:
This need to be backward compatible. Without specifying , it should by default use NB-DB.